src/main/java/com/hellosign/sdk/HelloSignClient.java

```diff
@@ -746,6 +745,28 @@ public File getFiles(String requestId, String format) throws HelloSignException
         return httpClient.withAuth(auth).withGetParam(PARAM_FILE_TYPE_URI, format).get(url).asFile(fileName);
     }
 
+    public String getFilesUrl(String requestId) throws HelloSignException {
+        String fileUrl = null;
+        String url = BASE_URI + SIGNATURE_REQUEST_FILES_URI + "/" + requestId;
+
+        HttpClient httpClient = this.httpClient.withAuth(auth).withGetParam(PARAM_TEMPLATE_GET_URL, "1").get(url);
```

Comment:
I would argue to say that `PARAM_TEMPALTE_GET_URL` could be renamed to `PARAM_GET_URL` it is the same value for retrieving file or template urls